.github/workflows/build.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
         uses: actions/checkout@v3
       - name: Setup LXD
         uses: canonical/setup-lxd@main
+        with:
+          channel: 5.20/stable
```

Comment:
what's special about this?

Reply:
There was a bug with newer versions of lxd being unable to unmount files from overlay when building the rock. I found this as a solution on other rock repos.